### ISSUE TITLE
(bug) useField/Field type checkbox:  Field and type checkbox doesnt support value since 6.3.1

### ISSUE THREAD

**Issue by vospascal** [final-form/react-final-form#707]
Since 6.3.1 (and higher) some how value is not passed to fields with a type checkbox any more.
When reverting to 6.3.0 it works again.

6.3.0 would print out {"name":"abc","value":true,"type":"checkbox","checked":true} when stringify is used on input. 

6.3.1 would print out {"name":"abc","checked":true,"type":"checkbox"} when stringify is used on input.

**Comment by vospascal** [comment]
example on https://codesandbox.io/s/relaxed-williams-32kek change version to see the difference

**Comment by vospascal** [comment]
@erikras is this something we can easly fix?

**Comment by dulakm** [comment]
I see that removing `type=checkbox` fixes the issue.

@erikras does it mean that we shouldn't specify type anymore? Shouldn't this be mentioned in release notes? Quite tricky bug to find when upgrading.

**Comment by vospascal** [comment]
but then you dont get the checked ...

**Comment by vospascal** [comment]
any update on this? looks like its still a breaking change @erikras

**Comment by erikras** [comment]
Can you set up a simpler sandbox? There's a lot of extra stuff going on in the one you provided that makes it hard to follow.

RFF provides two ways of using checkboxes:

1. `type="checkbox"` passed to `<Field/>`, managed value is a boolean. (like "Employed" on [Simple Example](https://final-form.org/docs/react-final-form/examples/simple))
2. `type="checkbox"` and `value="someString"` passed to `<Field/>`, managed value is an array of strings. (like "Sauces" on [Simple Example](https://final-form.org/docs/react-final-form/examples/simple)).

**Comment by vospascal** [comment]
never mind it was mine mistake.. ill close it
